addons/website/static/src/snippets/s_popup/000.scss

```diff
@@ -14,64 +14,51 @@
         display: block !important;
     }
 
-    $popup-close-size: 1.5rem;
     .s_popup_close {
-        position: absolute;
-        background: white;
-        height: $popup-close-size;
-        width: $popup-close-size;
-        line-height: $popup-close-size;
-        margin-top: -1 * $popup-close-size / 2;
-        margin-right: -1 * $popup-close-size / 2;
-        border-radius: $popup-close-size / 2;
-        right: 0px;
-        top: 0px;
-        box-shadow: rgba(0,0,0,0.8) 0 0 5px;
+        @include o-position-absolute(0, 0);
+        @include size($font-size-lg * 2);
+        line-height: $font-size-lg * 2;
+        @include o-bg-color(color-yiq(theme-color('primary')), theme-color('primary'));
+        box-shadow: $box-shadow-sm;
         cursor: pointer;
+        font-size: $font-size-lg;
         text-align: center;
-        z-index: 1;
-        font-size: $popup-close-size;
     }
 
-    &.s_popup_center {
-        .s_popup_full {
-            @include o-position-absolute(0, 0, 0, 0);
-            &.modal-dialog {
-                max-width: 100%;
-                padding: 0 !important;
-                margin: 0 !important;
-
-                .modal-content {
-                    height: 100%;
-                    width: 100%;
-                    justify-content: center;
+    .s_popup_full {
+        @include o-position-absolute(0, 0, 0, 0);
+        &.modal-dialog {
+            max-width: 100%;
+            padding: 0 !important;
+            margin: 0 !important;
+            .modal-content {
+                @include size(100%);
+                > section {
+                    display: flex;
+                    align-self: center;
+                    align-items: center;
+                    @include size(100%);
                 }
             }
-            .s_popup_close {
-                font-size: 60px;
-                margin: 10px;
-                background: none;
-                box-shadow: none;
-            }
         }
     }
 
     &.s_popup_fixed {
+        .s_popup_content {
+            position: fixed;
+            z-index: 1;
+            right: 0;
+        }
```

Comment:
It's just because it makes more sense to have this 'primary' rule first. I know it's not very compliant with the concept of 'minimum diff'. Do you want me to put it back to its initial position?

Reply:
Mmh, no you are right, I commented too quickly, it actually makes more sense indeed.